packages/static-module-record/test/test-static-module-record.js

```diff
@@ -68,35 +68,39 @@ function initialize(t, source, options = {}) {
 
   const compartment = new Compartment(endowments);
 
-  Object.keys(record.__liveExportMap__).forEach(name => {
-    /** @param {any} value */
-    const set = value => {
-      namespace[name] = value;
-      log.push(`${name}: ${JSON.stringify(value)}`);
-    };
-    const get = () => {
-      return namespace[name];
-    };
-
-    // Initialization uses the fast local variable.
-    liveUpdaters[name] = set;
-
-    // Live updates fall through to the scope proxy.
-    // Live updates could be accomodated with an invasive rewrite of all
-    // references to the variable in scope, but we elected to avoid
-    // transforming code except for import and export statements, in order to
-    // minimize surprise in debugging.
-    Object.defineProperty(compartment.globalThis, name, { get, set });
-  });
+  Object.entries(record.__liveExportMap__).forEach(
+    ([exportedName, [localName]]) => {
+      /** @param {any} value */
+      const set = value => {
+        namespace[exportedName] = value;
+        log.push(`${exportedName}: ${JSON.stringify(value)}`);
+      };
+      const get = () => {
+        return namespace[exportedName];
+      };
+
+      // Initialization uses the fast local variable.
+      liveUpdaters[localName] = set;
+
+      // Live updates fall through to the scope proxy.
+      // Live updates could be accomodated with an invasive rewrite of all
+      // references to the variable in scope, but we elected to avoid
+      // transforming code except for import and export statements, in order to
+      // minimize surprise in debugging.
+      Object.defineProperty(compartment.globalThis, localName, { get, set });
+    },
+  );
 
-  Object.keys(record.__fixedExportMap__).forEach(name => {
-    /** @param {any} value */
-    onceUpdaters[name] = value => {
-      t.assert(!(name in namespace));
-      namespace[name] = value;
-      log.push(`${name}: ${JSON.stringify(value)}`);
-    };
-  });
+  Object.entries(record.__fixedExportMap__).forEach(
+    ([exportedName, [localName]]) => {
+      /** @param {any} value */
+      onceUpdaters[localName] = value => {
```

Comment:
Likewise, what if the same `localName` is used in multiple fixed `exportedName`s?

Reply:
Oh, boy. I’m going to post a follow-up draft with a broken test and come back to it on the other side of some time away from keyboard.